### PR TITLE
Remove configuration

### PR DESCRIPTION
**Pull request by croaky**
Why:

* Since 0.5.0, Parity has been distributed as a binary, not a gem.
* It therefore does not pick up `Parity.configure`.
* It has been gone for months and no one has complained.

How it addresses the need:

* Remove the configuration.

https://github.com/thoughtbot/parity/issues/42

### CODE REVIEW

**Comment by gabebw** [comment]
I'm in favor of this change, for all the reasons you list, but especially:

> It has been gone for months and no one has complained.

**Comment by croaky** [comment]
Gonna let this sit a few days, see if there are any strong reactions.

**Comment by geoffharcourt** [comment]
I'm comfortable with going ahead and removing the config. In the event we find we absolutely need some kind of project-specific config, we should use the YAML config file approach to remain compatible with binaries.

File: spec/backup_spec.rb [hunk 1]
```diff
@@ -27,14 +24,18 @@
     Parity::Backup.new(
       from: "production",
       to: "staging",
-      additional_args: "--confirm thisismyapp-staging"
```
Comment (geoffharcourt) [inline spec/backup_spec.rb]: I don't think we want to remove this line. We still want to be able to send additional arguments to  commands even if we remove configuration.

**Comment by geoffharcourt** [comment]
@croaky, I think if you're comfortable this is good to merge.

**Comment by geoffharcourt** [comment]
Merged as 05ebb3c35d10627ed535ca97312f7b42471a9535.

**Pull request closed without merging**